android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -63,10 +63,10 @@
             getComments.setOnClickListener(v -> {
                 FeedItem clickedFeedItem = feedItems.get(getPosition());
                 commentItems = new ArrayList<>();
-                if(clickedFeedItem.getComments() != null) {
+                if (clickedFeedItem.getComments() != null) {
```

Comment:
While the formatting change you did was correct, but it is generally considered good practice to only change parts of code related to the issue you are solving. This makes it easier for reviewing and track changes.

If you want to fix the formatting, you can create another issue just for it and fix the formatting of entire project in a new PR :smile:

Reply:
@iamareebjamal 
i'll keep that in mind in the future. Actually i used a shortcut to change the formatting so it automatically changed the other code and it went unnoticed.
Should i change the formatting back to original?

Reply:
No leave it be for this PR